R/assertions.R

```diff
@@ -1482,9 +1482,11 @@ assert_expr_list <- function(arg, # nolint
 #'   e.g., `"Error in {arg_name}: the censor values must be zero."`.
 #'   If `message` argument is specified, that text will be displayed and `message_text`
 #'   is ignored.
-#' @param ... Objects required to evaluate the condition
-#'   If the condition contains objects apart from the element, they have to be
-#'   passed to the function. See the second example below.
+#' @param ... Objects required to evaluate the condition or the message text
```

Comment:
So is the traditional way to use `...`  .  I remember there was some discussion during argument renaming days that this was not standard convention??

Reply:
I feel like it is the standard way but I have never used this before - does it need so much flexibility?

Reply:
wait - now i am backtracking!